scala-stream-collector/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM java:8-alpine
+
+ENV SCALA_COLLECTOR=20170708 \
+BUILD_PACKAGES='bash curl-dev git curl ca-certificates gettext'
+
+WORKDIR /usr/local/scalacollector
```

Comment:
we might want to rename this to `scala-stream-collector`, in order to be coherent, what do you think?